django_project/localities_osm/utilities.py

```diff
@@ -1,8 +1,86 @@
 __author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
 __date__ = '04/04/19'
 
-from localities_osm.models.locality import LocalityOSMView
+from localities_osm.models.locality import LocalityOSMView, LocalityOSM
 
 
 def get_all_osm_query():
     return LocalityOSMView.objects.filter(osm_id__isnull=False).exclude(name='')
+
+
+def convert_into_osm_dict(locality):
+    """Utility to convert old locality dict into osm dict.
+
+    :param locality: Locality object.
+    :type locality: localities.models.Locality
+
+    """
+
+    data = locality.repr_dict()
+
+    osm_dict = data['values']
+    osm_dict['name'] = data['name']
+
+    if osm_dict.get('staff', None):
+        staff = osm_dict['staff'].split('|')
+        staff = [u'0' if v is u'' else v for v in staff]
```

Comment:
hi @ann26 please don't put 'staff_nurses' as 0 if not found
just keep osm_dict without staff_nurses